### PR TITLE
[de] Adding german translation of README.md in /de

### PR DESCRIPTION
**Pull request by 5k9m**

### CODE REVIEW

**Comment by bagder** [comment]
Thanks! Are you going to continue working on a German translation of the rest as well?

**Comment by 5k9m** [comment]
Yes, I am currently working on the next 2 chapters.
I have some time on my hands in the next two weeks; I hope I can cover quite a bit in this timeframe.

**Comment by JoschuaSchneider** [comment]
This is awesome, I'm looking forward to this!
Let me know if there is anything I can help you out with!

File: de/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Website
+
+Die Website von diesem Buch kann hier gefunden werden: [daniel.haxx.se/http3-explained](https://daniel.haxx.se/http3-explained).
+
+## Aushelfen
+
+Solltest du Irrtümer, fehlende Details, Fehler oder offensichtliche Lügen in diesem Dokument finden, dann sende uns bitte eine ausgebesserte Version des betroffenen Paragrafen und wir werden diese als neue Version veröffentlichen. Natürlich führen wir alle an, die aushelfen. Ich hoffe, dass wir das Dokument im Laufe der Zeit verbessern können.
+
+Fehler sollten als [Issues](https://github.com/bagder/http3-explained/issues) oder als [Pull Requests](https://github.com/bagder/http3-explained/pulls) auf der Github Seite des Buches übermittelt werden.
```
Comment (eliasp) [inline de/README.md]: GitHub's brand-name is officially camel-cased (`Github` → `GitHub`). Depends on whether you favor the German grammatically correct version of `Github` or the corporate identity compliant `GitHub`.

Comment (5k9m) [reply de/README.md]: Thanks! I think for the purpose of accuracy going with the official camel-cased name is better. I pushed the change to the latest pull request.

**Pull request merged**